src/pages/home/sidebar/OptionRowTitle/index.native.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/forbid-prop-types */
+import React, {memo} from 'react';
+import {
+    Text,
+} from 'react-native';
+import {propTypes, defaultProps} from './OptionRowTitleProps';
+
+
```

Comment:
Add comments to this file to explain why we need web and native versions. Is it just because native won't have tooltips, and if that's the case, doesn't the tooltip component already account for native devices?